org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/package.json

```diff
@@ -24,6 +25,8 @@
   "devDependencies": {
     "optimist": "*",
     "ava": "^0.15.2",
-    "es6-sleep": "*"
+    "es6-sleep": "*",
+    "opent2t-device-smartthingshub": "file:../../../org.opent2t.sample.hub.superpopular/com.smartthings.hub/test",
+    "opent2t-device-binaryswitch": "file:../../test"
```

Comment:
hmmm does this mean this will always need the test files even for say the CLI? how does this work ? should this be a test dependency and not a dev dependency?

---
In reply to: [90574294](https://github.com/openT2T/translators/pull/135#discussion_r90574294) [](ancestors = 90574294)